### ISSUE TITLE
Dll load Error occured while running Test console app

### ISSUE THREAD

**Issue by SaddamBInSyed** [sskodje/ScreenRecorderLib#79]
Hi @sskodje 

I have been using this screen record library since last year on my win 10 pc without any issues.

now I received a Windows 10 tablet version (hp brand) and when I tried to run the latest test_cosole_app it says 
"Unhandled Exception: System.IO.fileNotFoundException: Could not load file or assembly 'ScreenRecorderLib.dll or one of its dependencies. The specified module could not be found"

as mentioned in the readme.
I have installed VS 2015 C++ redistributable package and Media pack X64 pack for build 1511.

But no luck so far.

pleas advise what I missing here.

Windows os version details:
Windows 10 Home
Version 1511
OS build :  10586.1176
64 bit OS

**Comment by sskodje** [comment]
Usually such problems is due to a mismatch between exe and dll, such as exe is 32 bit and dll 64 bit or vice versa. I'd check the build configuration in visual studio that the library matches with the test app.

**Comment by SaddamBInSyed** [comment]
@sskodje 
Thanks for the reply

I have installed the X86 version of the VS2015 C++ redistributable package and now app works fine.
thank you.

**Comment by anditriendl** [comment]
A small update: I have noticed that, after installation of Visual Studio Community 2019 on the test machine, the demo project started working. So the missing dll / dependency / module has to be something that is automatically installed with Visual Studio.

**Comment by anditriendl** [comment]
@sskodje we are trying now to start from a fresh win10 install, and to step by step install components until the test application works. So far we have installed .NET runtime and VC redistributables, but none of these helped. Now we are out of ideas what else to try. 

So my question: could you name a few components that we could try to install to see if it works then?

**Comment by kmaki565** [comment]
In my project, we let user install VC redist from https://aka.ms/vs/16/release/vc_redist.x64.exe (as we build the project as x64) and nobody complaints. It's a bit different from the one linked to the readme, isn't it?

**Comment by sskodje** [comment]
Oh snap, i think you are right. Try

https://aka.ms/vs/16/release/vc_redist.x64.exe
or
https://aka.ms/vs/16/release/vc_redist.x86.exe

I'm sorry, and i'll update the readme straight away.

**Comment by anditriendl** [comment]
I tried the vc_redist links you have sent and again set them up on my freshly installed test VM - still did not work. Do you have any other idea what I could try to install?

@sskodje if you are interested to check it out yourself, I could share my screen or also I could send you RDP access to the test machine.

@kmaki565 I do not know if your product is free / freely available, but if you could share it with, I would happily try it out on my test VM.

**Comment by anditriendl** [comment]
@sskodje @kmaki565 sorry guys, I have to apologize. I found the problem and it was my fault: I deployed a **debug build** to the test VM which did not work without Visual Studio installed. I built now a x86 as well as a x64 **release** version and both worked.

This probably does not work because the project contains native code. I did test & run debug versions of pure .NET applications many times on machines without VS installed.

So, sorry again and thanks for the help!
